src/code/graph.c

```diff
@@ -12,15 +12,15 @@ FaultClient sGraphUcodeFaultClient;
 
 // clang-format off
 UCodeInfo D_8012D230[3] = {
-    { UCODE_F3DZEX, D_80155F50 },
+    { UCODE_F3DZEX, gspF3DZEX2_NoN_PosLight_fifoTextStart },
     { UCODE_UNK, NULL },
-    { UCODE_S2DEX, D_80113070 },
+    { UCODE_S2DEX, gspS2DEX2d_fifoTextStart },
```

Comment:
`gspS2DEX2d`? where is the last `d` from

Also in general, the microcode names here are based on the "vanilla" names (e.g. f3dex2 variants) I guess, and by their capacity (e.g. NoN no near clipping however that may have been established that this is what this ucode does)
Or are there strings inside the ucode that tell what version it is? I forgot, if I ever knew

Reply:
I explained where `d` and the other names come from in the PR description. The fact that this f3dex2 is a No-nearclipping, Point-Lighting-enabled ucode is clear from the string:
`RSP Gfx ucode F3DZEX.NoN  fifo 2.08J Yoshitaka Yasumoto/Kawasedo 1999` (`/Kawasedo` is the giveaway for point lights)

Reply:
Sorry for inability to read/remember

Is the choice/position of `PosLight` in `NoN_PosLight_fifo` arbitrary?

And then is `s2dex2d` made up from `s2dex2` + d(ebug) or actually called that anywhere

Reply:
I used the exact name from the AC map file, so it's the original order. `s2dex2d` is called exactly that in the unlinked sdk object